segmenters/nlp/NLTKSentencizer/__init__.py

```diff
@@ -0,0 +1,103 @@
+from typing import Dict, List
+
+from jina.executors.segmenters import BaseSegmenter
+
+
+class NLTKSentencizer(BaseSegmenter):
+    """
+    Segment text into sentences using :class:`nltk.PunktSentenceTokenizer`
+    for the specified language.
+
+    Example:
+
+    >>> sentencizer = NLTKSentencizer() # doctest:+ELLIPSIS
+    NLTKSentencizer@...[I]:post_init may take some time...
+    NLTKSentencizer@...[I]:NLTK english sentence tokenizer ready.
+    NLTKSentencizer@...[I]:post_init may take some time takes ... seconds (...s)
+    >>> text = "Today is a good day. Can't wait for tomorrow!"
+    >>> sentencizer.segment(text)
+    [{'text': 'Today is a good day.', 'offset': 0, 'location': [0, 20]}, {'text': "Can't wait for tomorrow!", 'offset': 1, 'location': [21, 45]}]
+
+
+    :param language: default='english'. Lowercased language name to initialize the sentence tokenizer, accepted languages are listed in :attr:`SUPPORTED_LANGUAGES`.
+    :type language: str
+    :param args: Additional positional arguments
+    :param kwargs: Additional keyword arguments
+    """
+
+    SUPPORTED_LANGUAGES = [
```

Comment:
is it a need to keep this exhaustive list?

Reply:
doesn't NLTK have a list of supported languages in their API?

Reply:
Their API is very unspecific about the supported languages.. I had added this to make errors more understandable.  The punkt tokenizer models are being downloaded from here: https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/index.xml , so I guess one way could be writing an XML parser to read and load this instead of keeping fixed names of languages.

But as I said, the main point of keeping this list was to make the error more understandable. It throws the same error if nltk_data folder is not found, if a language name is mis-spelt, or a non-existing model for a language is called. 

If you think it's more appropriate to delete it, I can definitely do that.